<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewFilm</class>
 <widget class="QWidget" name="AddNewFilm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>341</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="LEKino">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавление фильма</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>261</width>
     <height>21</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="LTitle">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdTitle"/>
  </widget>
  <widget class="QPushButton" name="BAdd">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="LDirector_name">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Режиссёр</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdDirecor_name"/>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="LGenre">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Жанр</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdGenre"/>
  </widget>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>261</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="LDate">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Дата выхода</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="EdDate"/>
  </widget>
  <widget class="QSplitter" name="splitter_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="LIMDb">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>IMDb рейтинг</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdIMDb"/>
  </widget>
  <widget class="QSplitter" name="splitter_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="LKinoP">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Кинопоиск</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdKinoP"/>
  </widget>
  <widget class="QSplitter" name="splitter_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="LStar">
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Кол-во звёзд</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EdStar"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
